demo/kubernetes/rook-storageclass.yaml

```diff
@@ -17,12 +17,9 @@ kind: StorageClass
 metadata:
    name: rook-block
    namespace: rook
-provisioner: kubernetes.io/rbd
+provisioner: rook.io/rookblock
```

Comment:
Yes i thought the same. But in Kubernetes the first part corresponds to the vendor and the second part correspond to the name of the plugin. I guess in this case is ok because the name of the vendor, rook.io, is also the name as the project. Ill change it.